### PR TITLE
feat(api): add Token.perm_manage_tokens and surrounding functionality

### PR DESCRIPTION
**Pull request by peterthomassen**
The migration sets `perm_manage_tokens = True` for all existing tokens,
and then sets the default to `False`.

Related: #347

---

Todos:
- docs update
- GUI update

It may also be worthwhile to discuss why I called the field `perm_manage_tokens` (I figured it would play along best with the other things we're going to add, but that we should talk about).

### CODE REVIEW

**Comment by peterthomassen** [comment]
Docs update and GUI adjustments are done. Ready for review!

File: docs/auth/tokens.rst [hunk 1]
```diff
@@ -3,77 +3,195 @@
...
+``perm_manage_tokens``
+    :Access mode: read, write
+    :Type: boolean
+
+    Permission to manage tokens (this one and also all others).  A token which
+    does not have this flag set cannot access the ``auth/tokens/`` endpoints.
+
+``token``
+    :Access mode: read-once
+    :Type: string (28 characters)
```
Comment (nils-wisiol) [inline docs/auth/tokens.rst:82]: Let's be not too specific with details here - this value could change in the future

File: webapp/src/components/Field/SwitchBox.vue [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    :required="required"
+    :rules="[v => !required || !!v || 'Required.']"
+    @change="input($event)"
+    @keyup="keyup($event)"
+  />
+</template>
+
+<script>
+export default {
+  name: 'Checkbox',
```
Comment (nils-wisiol) [inline webapp/src/components/Field/SwitchBox.vue:16]: typo

**Review by nils-wisiol — APPROVED** [review]
very nice. just two nits :)

**Review by nils-wisiol — APPROVED** [review]

File: docs/auth/tokens.rst (hunk 1 above)
Comment (peterthomassen) [reply docs/auth/tokens.rst:82]: done

File: webapp/src/components/Field/SwitchBox.vue (hunk 2 above)
Comment (peterthomassen) [reply webapp/src/components/Field/SwitchBox.vue:16]: done

**Pull request merged**